CHANGELOG.rst

```diff
@@ -5,11 +5,46 @@ Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
-[Unreleased]
-************
+Unreleased
+**********
+
+`1.0.0`_ (2021-07-22)
+*********************
 
 Added
 -----
 
+* Documentation
+  * Example notebooks
+  * Setup guide (`#120 <https://github.com/eWaterCycle/ewatercycle/issues/120>`_)
+  * HPC cluster guide
+* Forcing generation using `ESMValTool <https://www.esmvaltool.org/>`_ (`#28 <https://github.com/eWaterCycle/ewatercycle/issues/28>`_, `#87 <https://github.com/eWaterCycle/ewatercycle/issues/87>`_,)
+* Available parameter sets (`#118 <https://github.com/eWaterCycle/ewatercycle/issues/110>`_)
+* `PyMT <https://pymt.readthedocs.io/>`_ inspired interface for following models
+  * LISFLOOD
+  * MARRMoT M01 and M14
+  * PCR-GLOBWB
+  * wflow
+* Containerized models using `grpc4bmi <https://github.com/eWaterCycle/grpc4bmi>`_
+* Configuration files for system setup
+* Hydrograph plotting (`#54 <https://github.com/eWaterCycle/ewatercycle/issues/54>`_)
+* Typings
+* iso8601 time format (`#90 <https://github.com/eWaterCycle/ewatercycle/issues/90>`_)
+
```

Comment:
can we add util functions to the list here?

Reply:
functions in utils.py are mainly for internal use while CHANGELOG is for people use public interface of package. So I don't feel it is needed to add those.

However I could mention the extra non-pymt non-bmi methods for models.